### PR TITLE
Changes to support "ReplicasPerMaster" and "Instances"

### PR DESCRIPTION
**Pull request by MECHANDR**
## Description
- Added `ReplicasPerMaster` parameter in `New-AzRedisCache` cmdlet. 
- Added `ReplicasPerMaster` and `Instances` in the output of `Get-AzRedisCache` cmdlet.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RedisCache/RedisCache/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 1.3.0-preview
```
Comment (VeryEarly) [inline src/RedisCache/RedisCache/ChangeLog.md]: please put release note under upcoming release and remove line 22

**Comment by VeryEarly** [comment]
Hi @MECHANDR ,

please also add "Prerelease = 'preview'" in [Az.RedisCache.psd1](https://github.com/Azure/azure-powershell/blob/redis_2019_07_01_preview/src/RedisCache/RedisCache/Az.RedisCache.psd1)

File: src/RedisCache/RedisCache/ChangeLog.md (hunk 1 above)
Comment (MECHANDR) [reply src/RedisCache/RedisCache/ChangeLog.md]: Updated

**Comment by MECHANDR** [comment]
> Hi @MECHANDR ,
> please also add "Prerelease = 'preview'" in Az.RedisCache.psd1

Added. Thanks.

**Comment by VeryEarly** [comment]
Hi @MECHANDR ,

TestZoneRedundancy.json failed at CredScan, looks like some response bodies contain passwords, ca you please take a look?

**Pull request merged**